src/oca_github_bot/tasks/close_stale.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# Distributed under the MIT License (http://opensource.org/licenses/MIT).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from .. import github
+from ..config import switchable
+from ..github import gh_call, gh_datetime
+from ..queue import getLogger, task
+
+_logger = getLogger(__name__)
+
+
+INACTIVITY_PERIOD = relativedelta(years=1)
+LABEL_STALE = "stale 🗃"
+CLOSING_AS_STALE_COMMENT = (
+    "This PR had no activity for a long time. "
+    "It's therefore being closed as stale. 🤖"
```

Comment:
Wouldn't be suitable to also add something like "Please, reopen if this PR is still relevant"?

Reply:
:) That's so positive ))

Reply:
Better to be! People sometimes gets frustrated because it takes a lot of time to get something merged in OCA if you don't have reviews :sweat_smile:

Reply:
Yeah, politics

Reply:
Politics 🤷‍♂